### PR TITLE
RDM-4135: Media Viewer integration

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-4135

### Change description ###
Media viewer integration 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,7 +89,10 @@
     "url": "^0.11.0",
     "webdriver-manager": "^12.1.0",
     "x-frame-options": "^1.0.0",
-    "zone.js": "^0.8.26"
+    "zone.js": "^0.8.26",
+    "@hmcts/document-viewer-webcomponent": "^1.1.5",
```
Comment (MSancaktutar) [inline package.json]: are these dependencies actually needed (directly)?

File: angular.json [hunk 2]
```diff
@@ -27,13 +27,24 @@
...
+              {
+                "glob": "**/*",
+                "input": "node_modules/@hmcts/media-viewer/assets",
+                "output": "/assets"
               }
             ],
             "styles": [
               "src/style/app.scss",
-              "node_modules/govuk_template_mustache/assets/stylesheets/govuk-template.css"
+              "node_modules/govuk_template_mustache/assets/stylesheets/govuk-template.css",
```
Comment (MSancaktutar) [inline angular.json]: i wouldn't expect these to be needed directly here to use Media Viewer.

File: package.json (hunk 1 above)
Comment (MKaramath) [reply package.json]: Safely removed after analysing the impacts.

File: angular.json (hunk 2 above)
Comment (MKaramath) [reply angular.json]: Done.

**Comment by MSancaktutar** [comment]
Closing without merging

**Pull request closed without merging**